### ISSUE TITLE
Automated Accessibility Testing with Attest for Android

### ISSUE THREAD

**Issue by ljgoslin** [a11yproject/a11yproject.com#657]
This webinar is intended to teach developers how they can now trial Attest for Android for free and utilize the limited functionality framework in their own build. Our mobile accessibility expert, Chris McMeeking, will demonstrate how developers will now be able to trial Attest for Android simulator right on their desktops without having to request a demo.

[Join this webinar](https://accessibility.deque.com/automated-accessibility-testing-attest-for-android) on **Thursday, November 29th from 1:00-2:00PM EST**.

Our tool is fueled by our powerful Android dedicated rule engine, which means our accessibility testing is able to find and sort through accessibility issues that do not need to be resolved manually – providing zero false positives and 100% accuracy.

By downloading this free trial, developers will be able to:

Run Attest for Android on their application.
Save development time with analysis and debugging tools
Collaborate with screenshots that include View Information
View WCAG 2.0 AA violations in the interactive automated testing framework.
Learn accessibility best practices from the Deque Android pattern library rule sets examples.

**Comment by ericwbailey** [comment]
Closed via https://github.com/a11yproject/a11yproject.com/commit/dc7f5ea65b1d0998f95be45b7aa152dab71fb5ab
